build-config/make/crosstool-ng.make

```diff
@@ -9,16 +10,26 @@
 # This is a makefile fragment that defines the build of crosstool-NG
 #
 
+# Default GCC version to build for the toolchain
+GCC_VERSION 			?= 4.9.2
+XTOOLS_LIBC 			?= uClibc
+XTOOLS_LIBC_VERSION 		?= 0.9.33.2
+
 CROSSTOOL_NG_DESC		= crosstool-NG
-CROSSTOOL_NG_VERSION		= 1.21.0
-CROSSTOOL_NG_TARBALL		= crosstool-ng-$(CROSSTOOL_NG_VERSION).tar.xz
-CROSSTOOL_NG_URLS		+= $(ONIE_MIRROR) http://crosstool-ng.org/download/crosstool-ng
+CROSSTOOL_NG_VERSION		?= custom
```

Comment:
That was my original code in https://github.com/opencomputeproject/onie/pull/375 . But that would require separate directories for patches based on version in onie/patches/crosstool-NG; as well as separating the crosstool configurations based on crosstool version in conf/crosstool